src/test/java/com/fasterxml/jackson/databind/ser/TestRootType.java

```diff
@@ -67,6 +67,13 @@ static class TestCommandParent {
 
     static class TestCommandChild extends TestCommandParent { }
 
+    // [databind#4047]
```

Comment:
Although this is possible location, maybe rather add under `com.fasterxml.jackson.databind.node`; maybe in existing `TestConversions`?